### ISSUE TITLE
Add Gallery Section to AACoT'24 Conference Website

### ISSUE THREAD

**Issue by Mayaleeeee** [asyncapi/conference-website#264]
#### Description
The section should include a button labeled 'Browse All' or 'See All', and 

- will have a header titled 'Our Events Gallery'.
- a button labeled 'Browse All', and 

When users click on the button, they should be directed to a 

- Google Drive folder with all the images, 
**Or**  to a 
- new page with cards displaying the location, year, and a 'View Gallery' button for each event.

 Since AACoT'23 was our first in-person conference, there will be three cards (for each of the locations) with a 'View Gallery' button that leads to a Google Drive folder.

#### Expected Behavior
Users should be able to easily browse through the gallery of images from past events by clicking on the 'Browse All' or 'See All' button.

#### Why we should have this
This feature will enhance the user experience by providing a visual representation of past events.

**Comment by AceTheCreator** [comment]
Do you have any design in mind?

**Comment by Mayaleeeee** [comment]
> Do you have any design in mind?

Yeah, it's a minor thing. We can have it after the `About the Event AACoT'24.`

"We can use a Bento grid format for this. I like this idea!" Then we can have a button with `Browse All` or something similar
![WhatsApp Image 2024-03-07 at 1 31 52 PM](https://github.com/asyncapi/conference-website/assets/105395613/5255d86b-a4b4-4917-923a-4160233c47d3)

Or this 
![WhatsApp Image 2024-03-07 at 1 27 54 PM](https://github.com/asyncapi/conference-website/assets/105395613/9be41d97-7e64-4e26-85dd-58dee223defe)

**Comment by Oloso-surur** [comment]
I I'm interested to work on this issue. cc @Mayaleeeee @AceTheCreator

**Comment by Oloso-surur** [comment]
Here is the design I worked on.
![IMG-20240321-WA0000](https://github.com/asyncapi/conference-website/assets/143710076/422e340c-df0c-4773-bd3e-5c41f829896d)

**Comment by cybraia** [comment]
Hi @Oloso-surur are you working on the coding this up? If not, I would like to work on this. @Mayaleeeee do you want the this section just above the footer in the homepage?

**Comment by cybraia** [comment]
Hi @Mayaleeeee I added an events gallery section. Here is the commit [d7ef192](https://github.com/asyncapi/conference-website/pull/287/commits/d7ef1921de8cf0161d284e745ab9f88f3a08c5a3)

**Comment by thulieblack** [comment]
I see two PRs here. We assign issues on a first come basis; @oloso-surur, do you have a PR for this issue?

**Comment by Oloso-surur** [comment]
> I see two PRs here. We assign issues on a first come basis; @Oloso-surur, do you have a PR for this issue?

Nope, it's just this issue. I'm contributing as a designer.

Is there a problem?

**Comment by AceTheCreator** [comment]
@Oloso-surur you mentioned working on the design for this issue, do you mind sharing what you've got?

**Comment by Oloso-surur** [comment]
@AceTheCreator I already shared it on the comment above .

**Comment by anuragnegi000** [comment]
I would like to work into this

**Comment by anuragnegi000** [comment]
@Mayaleeeee can i add design by myself in code? or do you guys have any existing design to work on?

**Comment by anuragnegi000** [comment]
> > Do you have any design in mind?
> 
> Yeah, it's a minor thing. It can be placed after the `About the Event AACoT'24.`
> 
> **We can use a Bento grid format for this. I like this idea! Then we can have a button with `Browse All` or something similar** ![WhatsApp Image 2024-03-07 at 1 31 52 PM](https://private-user-images.githubusercontent.com/105395613/314401642-5255d86b-a4b4-4917-923a-4160233c47d3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxNjQyLTUyNTVkODZiLWE0YjQtNDkxNy05MjNhLTQxNjAyMzNjNDdkMy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjI1NDAxMDkxMjdlMGQ1N2Q4ODA4MTY5ZDU1M2UwNDk3OTk5M2RlNjlmNmFiMzM4NGYzNjc0NDZmNmVlZjJmZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oSxxx3GbsiL0Z7osZUGmzf7egGv_OhaAJCXXGQewBJ0)
> 
> Or this ![WhatsApp Image 2024-03-07 at 1 27 54 PM](https://private-user-images.githubusercontent.com/105395613/314401381-9be41d97-7e64-4e26-85dd-58dee223defe.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxMzgxLTliZTQxZDk3LTdlNjQtNGUyNi04NWRkLTU4ZGVlMjIzZGVmZS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGZiNDlhMDk0OTg2NmJhMDFhMDE4NWFmOWFjNWZkNWExNDQ0ZjFiYzJjN2E4MzUwMzQ1ZDA2ZjUwZTM4NmYyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.if_SvvD7eZxB2i2zBW3034yJ01cJ7uWP5gThH08AnxI)
> 
> cc @AceTheCreator

As i can see you have already shared the design above so should i start working into this?

**Comment by anuragnegi000** [comment]
> > > Do you have any design in mind?

@Mayaleeeee please give me a confirmation regarding design so i can proceed further

**Comment by AraAriyo** [comment]
I'm interested in working on this issue regarding the gallery design @AceTheCreator @Mayaleeeee

**Comment by AceTheCreator** [comment]
> I'm interested in working on this issue regarding the gallery design @AceTheCreator @Mayaleeeee

Please, go ahead. We need it 🙏🏾

**Comment by anuragnegi000** [comment]
@ashmit-coder

**Comment by Mayaleeeee** [comment]
> > > Do you have any design in mind?
> > 
> > 
> > Yeah, it's a minor thing. It can be placed after the `About the Event AACoT'24.`
> > **We can use a Bento grid format for this. I like this idea! Then we can have a button with `Browse All` or something similar** ![WhatsApp Image 2024-03-07 at 1 31 52 PM](https://private-user-images.githubusercontent.com/105395613/314401642-5255d86b-a4b4-4917-923a-4160233c47d3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxNjQyLTUyNTVkODZiLWE0YjQtNDkxNy05MjNhLTQxNjAyMzNjNDdkMy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjI1NDAxMDkxMjdlMGQ1N2Q4ODA4MTY5ZDU1M2UwNDk3OTk5M2RlNjlmNmFiMzM4NGYzNjc0NDZmNmVlZjJmZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oSxxx3GbsiL0Z7osZUGmzf7egGv_OhaAJCXXGQewBJ0)
> > Or this ![WhatsApp Image 2024-03-07 at 1 27 54 PM](https://private-user-images.githubusercontent.com/105395613/314401381-9be41d97-7e64-4e26-85dd-58dee223defe.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxMzgxLTliZTQxZDk3LTdlNjQtNGUyNi04NWRkLTU4ZGVlMjIzZGVmZS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGZiNDlhMDk0OTg2NmJhMDFhMDE4NWFmOWFjNWZkNWExNDQ0ZjFiYzJjN2E4MzUwMzQ1ZDA2ZjUwZTM4NmYyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.if_SvvD7eZxB2i2zBW3034yJ01cJ7uWP5gThH08AnxI)
> > cc @AceTheCreator
> 
> As i can see you have already shared the design above so should i start working into this?

Yes, please @anuragnegi000 
you can go ahead and add the design in code.

**Comment by Mayaleeeee** [comment]
> > I'm interested in working on this issue regarding the gallery design @AceTheCreator @Mayaleeeee
> 
> Please, go ahead. We need it 🙏🏾

The design part has been completed by @Oloso-surur. cc @AceTheCreator 

 or is @AraAriyo contributing as a developer?

**Comment by AceTheCreator** [comment]
> > > I'm interested in working on this issue regarding the gallery design @AceTheCreator @Mayaleeeee
> > 
> > 
> > Please, go ahead. We need it 🙏🏾
> 
> The design part has been completed by @Oloso-surur. cc @AceTheCreator
> 
> or is @AraAriyo contributing as a developer?

Nothing on this issue says that @Mayaleeeee :)

**Comment by anuragnegi000** [comment]
> > > > Do you have any design in mind?
> > > 
> > > 
> > > Yeah, it's a minor thing. It can be placed after the `About the Event AACoT'24.`
> > > **We can use a Bento grid format for this. I like this idea! Then we can have a button with `Browse All` or something similar** ![WhatsApp Image 2024-03-07 at 1 31 52 PM](https://private-user-images.githubusercontent.com/105395613/314401642-5255d86b-a4b4-4917-923a-4160233c47d3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxNjQyLTUyNTVkODZiLWE0YjQtNDkxNy05MjNhLTQxNjAyMzNjNDdkMy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjI1NDAxMDkxMjdlMGQ1N2Q4ODA4MTY5ZDU1M2UwNDk3OTk5M2RlNjlmNmFiMzM4NGYzNjc0NDZmNmVlZjJmZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oSxxx3GbsiL0Z7osZUGmzf7egGv_OhaAJCXXGQewBJ0)
> > > Or this ![WhatsApp Image 2024-03-07 at 1 27 54 PM](https://private-user-images.githubusercontent.com/105395613/314401381-9be41d97-7e64-4e26-85dd-58dee223defe.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxMzgxLTliZTQxZDk3LTdlNjQtNGUyNi04NWRkLTU4ZGVlMjIzZGVmZS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGZiNDlhMDk0OTg2NmJhMDFhMDE4NWFmOWFjNWZkNWExNDQ0ZjFiYzJjN2E4MzUwMzQ1ZDA2ZjUwZTM4NmYyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.if_SvvD7eZxB2i2zBW3034yJ01cJ7uWP5gThH08AnxI)
> > > cc @AceTheCreator
> > 
> > 
> > As i can see you have already shared the design above so should i start working into this?
> 
> Yes, please @anuragnegi000 you can go ahead and add the design in code.

I am starting to implement the code of given design

**Comment by AceTheCreator** [comment]
> > > > > Do you have any design in mind?
> > > > 
> > > > 
> > > > Yeah, it's a minor thing. It can be placed after the `About the Event AACoT'24.`
> > > > **We can use a Bento grid format for this. I like this idea! Then we can have a button with `Browse All` or something similar** ![WhatsApp Image 2024-03-07 at 1 31 52 PM](https://private-user-images.githubusercontent.com/105395613/314401642-5255d86b-a4b4-4917-923a-4160233c47d3.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxNjQyLTUyNTVkODZiLWE0YjQtNDkxNy05MjNhLTQxNjAyMzNjNDdkMy5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZjI1NDAxMDkxMjdlMGQ1N2Q4ODA4MTY5ZDU1M2UwNDk3OTk5M2RlNjlmNmFiMzM4NGYzNjc0NDZmNmVlZjJmZSZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.oSxxx3GbsiL0Z7osZUGmzf7egGv_OhaAJCXXGQewBJ0)
> > > > Or this ![WhatsApp Image 2024-03-07 at 1 27 54 PM](https://private-user-images.githubusercontent.com/105395613/314401381-9be41d97-7e64-4e26-85dd-58dee223defe.jpeg?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA3ODA1OTYsIm5iZiI6MTcyMDc4MDI5NiwicGF0aCI6Ii8xMDUzOTU2MTMvMzE0NDAxMzgxLTliZTQxZDk3LTdlNjQtNGUyNi04NWRkLTU4ZGVlMjIzZGVmZS5qcGVnP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MDcxMiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDA3MTJUMTAzMTM2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9OGZiNDlhMDk0OTg2NmJhMDFhMDE4NWFmOWFjNWZkNWExNDQ0ZjFiYzJjN2E4MzUwMzQ1ZDA2ZjUwZTM4NmYyMCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QmYWN0b3JfaWQ9MCZrZXlfaWQ9MCZyZXBvX2lkPTAifQ.if_SvvD7eZxB2i2zBW3034yJ01cJ7uWP5gThH08AnxI)
> > > > cc @AceTheCreator
> > > 
> > > 
> > > As i can see you have already shared the design above so should i start working into this?
> > 
> > 
> > Yes, please @anuragnegi000 you can go ahead and add the design in code.
> 
> I am starting to implement the code of given design

I can't view the images shared in this discussion 🤷🏾‍♂️

**Comment by Mayaleeeee** [comment]
Hey @anuragnegi000 
We'll need to change the colours @Oloso-surur used for the button.

**Comment by Mayaleeeee** [comment]
![Ace](https://github.com/user-attachments/assets/71453a2b-6a03-4110-866a-20363911af45)
@AceTheCreator

**Comment by anuragnegi000** [comment]
![eventGallery](https://github.com/user-attachments/assets/42a3df0d-1f1c-4418-a9b6-1ef8c7c4b96b)

@Mayaleeeee this is what i have created for now, please tell me if you need any changes in this

**Comment by thulieblack** [comment]
@anuragnegi000 please open a PR so its easy for people to review

**Comment by Saurabhchaudhary9799** [comment]
hey @AceTheCreator @thulieblack @Mayaleeeee  can you assign to me this issue , i want to contribute on this issue.

**Comment by anuragnegi000** [comment]
i am working on this : ) @Saurabhchaudhary9799

**Comment by techmannih** [comment]
@AceTheCreator  can I work on this issue?

**Comment by ashmit-coder** [comment]
> @AceTheCreator can I work on this issue?

Yes you can @techmannih as  @anuragnegi000 has not been responding. So go ahead!

**Comment by techmannih** [comment]
@ashmit-coder sure, Thank you!

**Comment by techmannih** [comment]
@ashmit-coder @Mayaleeeee  please check once 
![image](https://github.com/user-attachments/assets/1221442d-4a1e-426b-8d57-2192ab44a6d6)

**Comment by Mayaleeeee** [comment]
> hey @ashmit-coder , as discussed on Slack I have attached the video of my proposed design(responsive), and after reviewing this issue and previous pr for this, few question 
> 1. OnClick the browse button, it should redirect to Google Drive or a new page.
> 2. In the video I have used some generic images, can anyone point me to the images that should be used in this gallery component?
> Please let me know if I should make any changes to this component or raise a PR about what I have done.
> 
> 
> https://github.com/user-attachments/assets/37dcc9ec-4a8a-4746-98b5-1d646d165959
> 
> 

This looks good @ashmit-coder 
Is there anything else needed?

**Comment by ashmit-coder** [comment]
> > hey [@ashmit-coder](https://github.com/ashmit-coder) , as discussed on Slack I have attached the video of my proposed design(responsive), and after reviewing this issue and previous pr for this, few question
> > 
> > 1. OnClick the browse button, it should redirect to Google Drive or a new page.
> > 2. In the video I have used some generic images, can anyone point me to the images that should be used in this gallery component?
> >    Please let me know if I should make any changes to this component or raise a PR about what I have done.
> > 
> > 
> >   
> >     
> >     
> > 
> >     gallery.mp4
> 
> This looks good [@ashmit-coder](https://github.com/ashmit-coder) Is there anything else needed?

I don't think so @Mayaleeeee if the design is fine by you, then lets proceed.

**Comment by Mayaleeeee** [comment]
> > > hey [@ashmit-coder](https://github.com/ashmit-coder) , as discussed on Slack I have attached the video of my proposed design(responsive), and after reviewing this issue and previous pr for this, few question
> > > 
> > > 1. OnClick the browse button, it should redirect to Google Drive or a new page.
> > > 2. In the video I have used some generic images, can anyone point me to the images that should be used in this gallery component?
> > >    Please let me know if I should make any changes to this component or raise a PR about what I have done.
> > > 
> > > 
> > >   
> > >     
> > >     
> > > 
> > >     gallery.mp4
> > 
> > This looks good [@ashmit-coder](https://github.com/ashmit-coder) Is there anything else needed?
> 
> I don't think so @Mayaleeeee if the design is fine by you, then lets proceed.

It looks good 👍 👌

**Comment by rohithyarramala** [comment]
@Mayaleeeee  is the issues still exists !

**Comment by ashmit-coder** [comment]
> [@Mayaleeeee](https://github.com/Mayaleeeee) is the issues still exists !

Hey @rohithyarramala @om2137 is working on it already!

**Comment by Recxsmacx** [comment]
@AceTheCreator are you guys open for some design suggestions or it is finalized?

**Comment by Mayaleeeee** [comment]
> > [@Mayaleeeee](https://github.com/Mayaleeeee) is the issues still exists !
> 
> Hey @rohithyarramala @om2137 is working on it already!

Hey @rohithyarramala @om2137 are you still working on this?

**Comment by om2137** [comment]
> > > [@Mayaleeeee](https://github.com/Mayaleeeee) is the issues still exists !
> > 
> > Hey @rohithyarramala @om2137 is working on it already!
> 
> Hey @rohithyarramala @om2137 are you still working on this?

@Mayaleeeee yes I have raised the PR waiting for your review. 
@AceTheCreator and @ashmit-coder have already reviewed it.

**Comment by harshhgithub** [comment]
hii @Mayaleeeee is this issue still open , can i work on it ?

**Comment by HarshitaJaiswal5** [comment]
Hey @Mayaleeeee I'm interested to work on this issue. Could you assign it to me ?
